lib/senders/index.js

```diff
@@ -4,16 +4,15 @@
 
 'use strict'
 
-var P = require('../promise')
 // This indirection exists to accommodate different config properties
 // in the old auth mailer. If/when the two config files are merged and
 // there's nothing left that imports mailer/config, it is safe to merge
 // raw.js and this file into one. Be careful not to mix the arguments
 // up when you do that, they expect config and log in a different order.
-var createSenders = require('./legacy_index')
+const createSenders = require('./legacy_index')
```

Comment:
Everything under `lib/senders` needs to remain as valid ES5 syntax for static analysis by the l10n scripts, until we update the l10n stuff to use an ES6 aware parser. So you'll need to revert the ES6 stuff in here, sorry.

Note that there's some ES6 stuff we can get away with still because it's only the static analysis requirement that we have to satisfy, e.g. see the usage of `Map` in `lib/senders/sms.js`.